src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Altinn.App.PlatformServices.csproj

```diff
@@ -27,7 +27,7 @@
   <ItemGroup>
     <PackageReference Include="Altinn.Common.AccessTokenClient" Version="0.5.4" />
     <PackageReference Include="Altinn.Platform.Models" Version="1.0.4-alpha" />
-    <PackageReference Include="Altinn.Platform.Storage.Interface" Version="2.1.18" />
+    <PackageReference Include="Altinn.Platform.Storage.Interface" Version="2.2.0" />
     <PackageReference Include="HtmlAgilityPack" Version="1.11.24" />
     <PackageReference Include="JWTCookieAuthentication" Version="2.4.0-alpha" />
     <PackageReference Include="LibGit2Sharp" Version="0.26.2" />
```

Comment:
Is LibGit2Sharp really needed in this package?

Reply:
It compiles without so I removed it now.